docs/_documentations/container-registry-credentials.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: docs
+title: Container registry credentials for Codewind
+description: Specifying container registries for Codewind
+keywords: docker, containers, registry, Codewind, Appsody
+duration: 5 minutes
+permalink: containerregistry
+type: document
+parent: installoncloud
+order: 1
+---
+# Specifying Container Registries for Codewind 
+
+When you run Codewind on Kubernetes, Codewind uses [`buildah`](https://github.com/containers/buildah) to build container images from Dockerfiles and then push the images to a container registry that you specify.
+
+There are two scenarios in Codewind where you need to specify what container registry to use:
+
+1. If you are running in Che and creating a Codewind style project, you need to specify zero or one container registry so Codewind can push images to that specific registry. 
+2. If you are developing an Appsody style project, you need to specify zero or more container registries for Appsody to pull stack images from that specific registry. 
```

Comment:
It might be better to say something like `If you are using Codewind running on a remote Kubernetes cluster to develop an Appsody style project and the image registry for the Appsody stack requires credentials, you will need to configure Codewind to use the credentials for the registry`

There is also a third scenario, which was added/discovered late last week, `If you wish to use Codewind running locally to develop an Appsody style project and the image registry for the Appsody stack requires credentials, you will need to 'docker login' to the registry from the command line before creating the Appsody project`